app/lib/govuk/markdown_renderer.rb

```diff
@@ -0,0 +1,36 @@
+module Govuk
```

Comment:
We should probably look to move this to a gem its already being used on PUBLISH, FIND, APPLY and now here